.github/workflows/python-package-conda.yml

```
@@ -38,12 +38,16 @@ jobs:
         run: |
           pytest
       - name: Pytest Coverage
-        if: ${{ !github.event.pull_request.head.repo.fork }}
         run: |
-          pytest --cov=credit tests/ | tee pytest-coverage.txt
-      - name: Pytest coverage comment
-        uses: MishaKav/pytest-coverage-comment@1bdbba85fb74a2fdc1ec66383acec1091610f82b # v1.1.57
-        if: ${{ !github.event.pull_request.head.repo.fork }}
+           pytest --cov --cov-branch --cov-report=xml
+        #  pytest --cov=credit tests/ | tee pytest-coverage.txt
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v5
```

```suggestion
        uses: codecov/codecov-action@5a1091511ad55cbe89839c7260b706298ca349f7 # v5.5.1
```

Just pinning this to the commit SHA like the other third party actions.